arch/bootstrap/base.py

```diff
@@ -461,7 +461,7 @@ def conf_int(self, func, reps=1000, method='basic', size=0.95, tail='two',
                                   'too small to use BCa'
                         raise RuntimeError(message.format(jk_var=denom))
                     a = numer / denom
-                    a = a[:, None]
+                    a = np.atleast_1d(a)
```

Comment:
I think you need both atleast_1d and [:,None].

Reply:
Tried both `np.atleast_1d(a)` and `np.atleast_1d(a)[:,]`, they produced the same result. [:,None] won't work as it caused the original error. 

Or am I missing something?

See debug output:

```
In [2]: %debug
> /usr/anaconda3/lib/python3.6/site-packages/arch/bootstrap/base.py(464)conf_int()
    462                         raise RuntimeError(message.format(jk_var=denom))
    463                     a = numer / denom
--> 464                     a = a[:, None]
    465                 else:
    466                     a = 0.0
IndexError: invalid index to scalar variable.

ipdb> a
self = Stationary Bootstrap(block size: 5, no. pos. inputs: 1, no. keyword inputs: 0, ID: 0x7fb0e04f8a90)
func = <function mean at 0x7fb0d8306b70>
reps = 1000
method = 'bca'
size = 0.95
tail = 'two'
extra_kwargs = None
reuse = False
sampling = 'nonparametric'
std_err_func = None
studentize_reps = 0
ipdb> np.atleast_1d(a)[:,]
array([ 0.00075776])
ipdb> np.atleast_1d(a)
array([ 0.00075776])
```